.github/ISSUE_TEMPLATE/contribute.txt

```diff
@@ -0,0 +1,24 @@
+---
+name: Contribute
+about: Let us know you have an example you'd like to contribute
+
+---
+
+**Description**
+A clear and concise description of the concepts in your project and the problem your code solves. Please include any references you used. 
+
+**How to run the example**
+A short description of how to run your code, including an example. 
+
+**Link to your repo**
+Include a link to the repo you want to contribute. We will fork it. If we want to make any changes to the code we will create a pull request for you to approve.
+
+**Environment**
+ - Python version: [e.g., 3.7.0]
+ - Ocean version: [e.g., 2.0]
+
+**Additional Context**
```

Comment:
"Additional information" (non-cap to be consistent with previous one) or "Context"?